src/components/Header/components/Nav.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import './Nav.scss';
+import NAV_LIST from '../navData';
+const Nav = ({ isActive, showNav }) => {
+  return (
+    <nav className={isActive ? 'nav active' : 'nav'}>
+      <div className="dim" onClick={showNav} />
+      <div className="nav_box">
```

Comment:
Nav.scss에서 남긴 리뷰 확인해주시고, 동적인 className을 nav_box에 적용하시면 될 것 같네요!
그렇다면 최상단의 nav는 단순히 감싸고 있는 부분이기 때문에 React.Fragment로 처리하셔도 될 것 같습니다.
그리고, 공통적으로 nav_box의 스타일링을 가지고 있고 active할 경우에 className을 부여하는 걸 나타내기 위해 다음처럼 공통적인 부분 제외하고 유동적으로 className을 부여할 수 있겠네요!
```suggestion
    <>
      <div className="dim" onClick={showNav} />
      <div className=`nav_box${isActive ? ' active' : ''}`>
```